docroot/themes/humsci/humsci_basic/humsci_basic.theme

```diff
@@ -8,6 +8,31 @@
 use Drupal\Component\Utility\Html;
 use Drupal\node\Entity\Node;
 
+/**
+ * Implements hook_preprocess_block().
+ */
+function humsci_basic_preprocess_block(&$vars) {
+  // The following code block (lines 16 - 30) is an adaptation of the code found within the su_humsci theme preprocess block. `docroot/themes/humsci/su_humsci_theme/su_humsci_theme.theme`
+
```

Comment:
No empty lines after a comment. also this comment line is too long. Code climate should have picked that up. This comment could go below the `Implements` in the docblock.